packages/safe-apps-developer-ui/src/components/pages/safes/apps/AppIframe.tsx

```diff
@@ -30,15 +30,20 @@ const AppIframe = ({ url, app }: { url: string; app: SafeApp }): React.ReactElem
   const iframeRef = React.useRef<HTMLIFrameElement>(null);
   const communicator = useAppCommunicator(iframeRef, app);
   const [proposedTxs, setProposedTxs] = React.useState<ProposedTxs | null>(null);
-  const [networkId, provider] = useProviderStore((state) => [state.networkId, state.provider]);
+  const [networkId, provider, chainId] = useProviderStore((state) => [
+    state.networkId,
+    state.provider,
+    state.networkId,
```

Comment:
Here the `chainId` is returned copying the `networkId` value. All the networks in the enum `ETHEREUM_NETWORK_TO_ID` have the same value for both, except MORDEN. MORDEN has `networkId = 2` but `chainId = 62` (as far as I can see here: https://chainid.network/)